### ISSUE TITLE
Install NetHunter OnePlus 3T

### ISSUE THREAD

**Issue by Koploseus** [offensive-security/kali-nethunter#831]
Device: Oneplus 3T

OS version (KitKat/Lollipop/Marshmallow/Nougat): Nougat

 Built from repo (date and build command) or downloaded from website (links):
https://build.nethunter.com/nightly/3.15.4-20170202-1604/

## Issue:
Hello! I just bought the OnePlus 3T and i want to put NetHunter, the thing is that i read a lot of comments who says that there is a lot of problem etc etc.. i don't won't to break my phone. So how can i put nethunter on my phone ? I saw a lot of tutoriel but there is so many that i don't know witch one.

Sorry for my bad english i am french

Thanks!

**Comment by asomyhz** [comment]
Be root your dvice only 
or wait to news dvice support this app 
am arabic
my english is bad too

**Comment by f3l1x30** [comment]
It is difficult to "Break" your Oneplus 3T, I've formatted mine and reformatted it, flashed it with various different ROMs and it still works great. Nethunter works well on the Oneplus 3T. 

This is how to root: https://youtu.be/wwd31NU7VC0 but MAKE SURE you download and flash OxygenOs 4.0.2 not the latest version and DON'T let your device update it later. The Nethunter kernel for the Oneplus 3T is based on OxygenOs 4.0.2.

Then install Nethunter: https://forum.xda-developers.com/oneplus-3t/development/kernel-kali-nethunter-oneplus-3t-t3507816

Any questions, just ask :)

**Comment by zivoy** [comment]
@f3l1x30 I tried doing as you said but after I installed oxygen os 4.0.2 but, the phone didn't work I'm guessing it had compatibility issues because it turns on then it's on the unlock page for about 5 seconds and then turns off then it repeats that. but my stuff is still there as I can use TWRP to get back to oxygen os 5.0.5 and it works again. but it doesn't work on 4.0.2 so I can't install the kernel to work on it.
